src/mkdocstrings/handlers/__init__.py

```diff
@@ -110,20 +111,24 @@ def __init__(self, directory: str, theme: str) -> None:
         Arguments:
             directory: The name of the directory containing the themes for this renderer.
             theme: The name of theme to use.
+            custom_templates: Directory containing custom templates.
         """
+
+        self.directory = directory
```

Comment:
As stated in a previous comment, you can drop this line and use `handler_name` instead of `handler.renderer.directory`.

Reply:
Fixed in e275221